### ISSUE TITLE
System.NullReferenceExceptions blocking 10.8.0 stable

### ISSUE THREAD

**Issue by theAeon** [ShokoAnime/Shokofin#23]
Running latest unstable on the recently released jellyfin version, got sparse errors on library refresh. Noticed these two in particular:

```
System.NullReferenceException: Object reference not set to an instance of an object.
   at Shokofin.Providers.ExtraMetadataProvider.RemoveDuplicateEpisodes(Episode episode, String episodeId)
   at Shokofin.Providers.ExtraMetadataProvider.OnLibraryManagerItemUpdated(Object sender, ItemChangeEventArgs e)
   at Emby.Server.Implementations.Library.LibraryManager.UpdateItemsAsync(IReadOnlyList`1 items, BaseItem parent, ItemUpdateType updateReason, CancellationToken cancellationToken)
[2022-06-11 14:28:12.837 -04:00] [INF] Found episode for "Fate/Kaleid Liner Prisma Illya" (Series="171",Episode="3403",File="1898")
[2022-06-11 14:28:12.839 -04:00] [INF] Removing item, Type: "Episode", Name: "S2 World at Peace", Path: "/mnt/ryzenshokoshow/Durarara!!/Durarara!! - 00x02 - World at Peace.mkv", Id: c548788b-56d4-5857-bfb0-26848341c1d2
```

and

```
[2022-06-11 14:28:16.218 -04:00] [INF] Found episode for "Ansatsu Kyoushitsu (2015)" (Series="255",Episode="6047",File="3165")
[2022-06-11 14:28:17.005 -04:00] [ERR] Error in ItemUpdated event handler
System.NullReferenceException: Object reference not set to an instance of an object.
   at Shokofin.Providers.ExtraMetadataProvider.EpisodeExists(String episodeId, String seriesId, String groupId)
   at Shokofin.Providers.ExtraMetadataProvider.AddVirtualEpisode(GroupInfo groupInfo, SeriesInfo seriesInfo, EpisodeInfo episodeInfo, Season season)
   at Shokofin.Providers.ExtraMetadataProvider.UpdateSeries(Series series, String seriesId)
   at Shokofin.Providers.ExtraMetadataProvider.OnLibraryManagerItemUpdated(Object sender, ItemChangeEventArgs e)
   at Emby.Server.Implementations.Library.LibraryManager.UpdateItemsAsync(IReadOnlyList`1 items, BaseItem parent, ItemUpdateType updateReason, CancellationToken cancellationToken)
```

Some browsing of the changelog from beta 3 led me to https://github.com/jellyfin/jellyfin/pull/7867 and https://github.com/jellyfin/jellyfin/pull/7802 as possible culprits, but i'm not exactly familiar enough with the api to debug myself.

**Comment by revam** [comment]
Will look at it on Monday

**Comment by revam** [comment]
Fixed by the 2.0.0 release
